src/seam-connect/routes.ts

```diff
@@ -635,7 +636,7 @@ export abstract class Routes {
           }
         ),
       delete: (params: ClimateSettingScheduleDeleteRequest) =>
-        this.makeRequest({
+        this.makeRequest<ClimateSettingScheduleDeleteResponse>({
```

Comment:
We should probably do

```
        await this.makeRequest<ClimateSettingScheduleDeleteResponse>({
        ...
        return null
```

Reply:
[7f41a70](https://github.com/seamapi/javascript/pull/263/commits/7f41a70b01e2a469831fbf0b4a5cb9b3fb2b76bb)